server/commonTransactions/removeCellType/removeCellType.ts

```diff
@@ -0,0 +1,20 @@
+import FormWizard from 'hmpo-form-wizard'
+import { NextFunction, Response } from 'express'
+import FormInitialStep from '../../controllers/base/formInitialStep'
+
+export default class RemoveCellType extends FormInitialStep {
+  override saveValues(req: FormWizard.Request, res: Response, next: NextFunction) {
+    const { id: locationId } = res.locals.decoratedResidentialSummary.location
+    const { sessionModel } = req
+    const { cellId } = req.params
+    const normalCellTypes = `create-cells_set-cell-type_normalCellTypes${cellId}`
+    const specialCellTypes = `create-cells_set-cell-type_specialistCellTypes${cellId}`
+    const cellAccommodationType = `create-cells_set-cell-type_accommodationType${cellId}`
```

Comment:
This will work fine for now, but will break if this common transaction is used in any other transaction, I'll have a think about how to resolve it